sdk/servicebus/service-bus/src/core/linkEntity.ts

```diff
@@ -103,14 +143,15 @@ export class LinkEntity {
     this.address = options.address || "";
     this.audience = options.audience || "";
     this.name = getUniqueName(name);
+    this._logPrefix = `[${context.connectionId}|l:${this.name}|a:${this.address}]`;
```

Comment:
`l:${this.name}` - `l` refers to the link here?

Reply:
It does. @ramya-rao-a  had a good suggestion to include the type of link. I was thinking:

s == sender
br == batching receiver
sr == streaming receiver
(or maybe just 'r' for receiver?)
m == management link

I'm also going to pass in the logger which should ensure they get routed to the appropriate spot too.

Reply:
Nice!

Reply:
Added in. 

We already had a weird little situation with sessions but they were apparently using the logger.messageSession link so those are abbreviated as 'ms'.

I added a comment in as well as a little type for it (all internal).